packages/apps/graph/src/scripts/devnet/helper.ts

```diff
@@ -31,7 +31,7 @@ export const logger = createLogger('info');
 const getClient = (): IClient =>
   createClient(
     ({ chainId, networkId }) =>
-      `http://localhost:${devnetConfig.PORT}/chainweb/0.0/${networkId}/chain/${chainId}/pact`,
+      `http://127.0.0.1:${devnetConfig.PORT}/chainweb/0.0/${networkId}/chain/${chainId}/pact`,
```

Comment:
I seem to be having issues using localhost sometimes, i will likely revert this once I debug it further.